Cargo.lock

```diff
@@ -1022,9 +1022,9 @@ dependencies = [
 
 [[package]]
 name = "proc-macro2"
-version = "1.0.69"
+version = "1.0.78"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "134c189feb4956b20f6f547d2cf727d4c0fe06722b20a0eec87ed445a97f92da"
+checksum = "e2422ad645d89c99f8f3e6b88a9fdeca7fabeac836b1002371c4367c8f984aae"
```

Comment:
Can you change this back to .69 since it isn't necessary for this to select the newer version inside this repo?

Reply:
Shouldn't I remove the whole file instead?

Reply:
There shouldn't be any changes to the lock file, yup.